.gitignore

```diff
@@ -16,11 +16,11 @@ erl_crash.dump
 # Also ignore archive artifacts (built via "mix archive.build").
 *.ez
 
-# Don't include generated MAVLink file
-MAVLink.ex
```

Comment:
Superseded by today's changes, not creating hardcoded MAVLink.ex anymore and don't need to ignore.